build.psm1

```diff
@@ -168,9 +168,20 @@ cmd.exe /C cd /d "$location" "&" "$($vcPath)\vcvarsall.bat" "$Arch" "&" cmake "$
         $FilesToCopy = @('pwrshplugin.dll', 'pwrshplugin.pdb')
         $dstPath = "$PSScriptRoot\src\powershell-win-core"
         $FilesToCopy | ForEach-Object {
-            $srcPath = Join-Path (Join-Path (Join-Path (Get-Location) "bin") $Configuration) "CoreClr/$_"
+            $srcPath = [IO.Path]::Combine((Get-Location), "bin", $Configuration, "CoreClr/$_") 
+
             log "  Copying $srcPath to $dstPath"
             Copy-Item $srcPath $dstPath
+            
+            if ($_ -match "pwrshplugin.")
+            {
+                # Copy the plugin dll to the output directory so that the remoting tests can run out of the default build directory
+                $tempOptions = New-PSOptions -Configuration ($script:Options).Configuration -Framework ($script:Options).Framework -Runtime ($script:Options).Runtime
+                $pluginDst = [IO.Path]::GetDirectoryName($tempOptions.Output) # skip the powershell.exe that gets added on the end
+                New-Item -Type Directory $pluginDst -Force
+                log "  Copying $srcPath to $pluginDst"
+                Copy-Item $srcPath $pluginDst
```

Comment:
I don't think we need this change. We now have the NuGet package [psrp.windows](https://powershell.myget.org/feed/powershell-core/package/nuget/psrp.windows) which contains `'pwrshplugin.dll', 'pwrshplugin.pdb' and 'Install-PowerShellRemoting.ps1'`. This NuGet package is referenced in `src\powershell-win-core`, so those 3 files will be published to the publish folder after `Start-PSBuild`. If changes are made to the native code, then a new psrp.windows package will be published.